plugins/modules/activation_key.py

```diff
@@ -45,6 +45,12 @@
     description:
       - Name of the content view
     type: str
+  content_view_environments:
+    description:
+      - List of content view environments to add to activation key
+      - Ignored if content_view_id and lifecycle_environment_id are specified
```

Comment:
```suggestion
      - Ignored if C(content_view) and C(lifecycle_environment) are specified
```
`C()` makes nice formatting, and the params are called without the `_id` here.

But this made me wonder: we can enforce mutual exclusivity in Ansible, so the user gets an error when they pass `content_view` *and* `content_view_environments`. Should we do that?

Then the description here would become something like
```suggestion
      - Mutually exclusive with C(content_view) and C(lifecycle_environment).
```

And around line 280 we'd need to add something like
```python
mutually_exclusive=[
  ['content_view', 'content_view_environment'],
  ['lifecycle_environment', 'content_view_environment']
]
```

Reply:
> But this made me wonder: we can enforce mutual exclusivity in Ansible, so the user gets an error when they pass content_view and content_view_environments. Should we do that?

TBH, I just copied the behavior of the API/hammer one-to-one. content-view-envionments will be ignored if you specify a single content-view/lifecycle-environment. No errors. I don't like that too....

The question is: should the ansible collection behave like the API or should it be better? :)

Reply:
I think we're allowed to be better ;-)

(Although I will poke @jeremylenz and @parthaa, as I think API and hammer could/should be stricter here too)